examples/Kucoin/Futures.jl

```diff
@@ -0,0 +1,31 @@
+# Kucoin/Futures
+# https://docs.kucoin.com/
+
+using CryptoAPIs
+using Dates, NanoDates
+using CryptoAPIs.Kucoin
```

Comment:
```suggestion
using Dates
using CryptoAPIs
using CryptoAPIs.Kucoin
```
use the DateTime type for queries